src/back-to-top/index.scss

```diff
@@ -0,0 +1,21 @@
+.back-to-top {
+  background: $white;
+  padding: 0 10px;
+
+  position: sticky;
```

Comment:
This is _quite_ badly supported cross browser. Have you tested in any not-latest-Chrome browsers?

https://caniuse.com/#feat=css-sticky